packages/core/src/lambda/vue/configEditor/samInvokeFrontend.ts

```diff
@@ -170,7 +172,7 @@ export default defineComponent({
 
             const source = this.resourceData?.source
 
-            client.invokeLaunchConfig(config, source).catch((e: Error) => {
+            client.invokeLaunchConfig(config, source, this.useDebugger).catch((e: Error) => {
```

Comment:
just set `config.noDebug`. you don't need a new `this.useDebugger` field.

Reply:
`config` here doesn't have a `noDebug` property, so we need to pass `this.useDebugger` to be able it to use it in `vscode.debug.startDebugging()`. 

Function definition
```
function debug.startDebugging(folder: vscode.WorkspaceFolder | undefined, nameOrConfiguration: string | vscode.DebugConfiguration, parentSessionOrOptions?: vscode.DebugSession | vscode.DebugSessionOptions): Thenable<boolean>
```
`vscode.DebugSessionOptions` also has a `noDebug` field, which is what we're using in this case.

Reply:
> `config` here doesn't have a `noDebug` property

that's just an oversight in this definition: https://github.com/aws/aws-toolkit-vscode/blob/db673c9b74b36591bb5642b3da7d4bc7ae2afaf4/packages/core/src/lambda/vue/configEditor/samInvokeFrontend.ts#L422

Reply:
Updated.